api/handlers/exporter.go

```diff
@@ -0,0 +1,102 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spectypes "github.com/ssvlabs/ssv-spec/types"
+
+	"github.com/ssvlabs/ssv/api"
+	exporterapi "github.com/ssvlabs/ssv/exporter/api"
+	"github.com/ssvlabs/ssv/exporter/convert"
+	ibftstorage "github.com/ssvlabs/ssv/ibft/storage"
+)
+
+type Exporter struct {
+	DomainType spectypes.DomainType
+	QBFTStores *ibftstorage.QBFTStores
+}
+
+type ParticipantResponse struct {
+	Role      string `json:"role"`
+	Slot      uint64 `json:"slot"`
+	PublicKey string `json:"public_key"`
+	Message   struct {
+		Signers []uint64 `json:"Signers"`
+	} `json:"message"`
+}
+
+func (e *Exporter) Decideds(w http.ResponseWriter, r *http.Request) error {
+	var request struct {
+		From    int           `json:"from"`
```

Comment:
Just something to think of: if 'from' or 'to' are not passed, the method will use zero values and try to fetch decided messages for slot 0, which might not be clear. However, passing 0 to both 'from' and 'to' looks fine to me

Reply:
Also, I think 'from' and 'to' should be better `uint64`s, same as `phase0.Slot`